### ISSUE TITLE
422 Invalid Parameters

### ISSUE THREAD

**Issue by bretthandrews** [sdss/marvin#258]
I clicked on a galaxy (7962-6103) in the image roulette page, and I got a 422 when it tried to go to the galaxy page. This error might have been caused by selecting MPL-4. This failure occurred in Chrome. When I switched to Firefox (and was auto-directed to MPL-5), I was able to successfully go to the galaxy page for 7962-6103 (via navbar search box). When I switched that session to MPL-4 and reloaded the page I got a 422. However, when I tried in the same Chrome session to switch to MPL-5 and go to the galaxy page via navbar search box, it returned a 422.

The validation error says that the parameter `bintemp` is `Not a valid choice`. I'm not sure that this is a helpful message.

FWIW I can get the cube and maps remotely via the tools.

js console error: Failed to load resource: the server responded with a status of 422 (UNPROCESSABLE ENTITY)

Chrome (MPL-4)
![screen shot 2017-05-10 at 9 42 46 am](https://cloud.githubusercontent.com/assets/12723826/25901566/2915d4be-3565-11e7-881f-baf8740a6333.png)

Safari (MPL-4):
![screen shot 2017-05-10 at 10 18 00 am](https://cloud.githubusercontent.com/assets/12723826/25903302/08416ad2-356a-11e7-8793-f9616906cea9.png)

**Comment by havok2063** [comment]
Yeah, the 422 status code error is what you get when your argument validation fails.  I will have to check and see if I can reproduce the error, and see if it's general or not.  Not sure how much control I have about the message.  That's mostly dynamically generated.

**Comment by havok2063** [comment]
I just tried reproducing this.  I was able to reproduce it on all browsers on the main site.  I tried reproducing this on the test server running off master, with all three browsers, same galaxy, and I could not.  I toggled back and forth between MPL-4 and MPL-5 and the page loaded ok.  So maybe this got fixed between the patch and the latest master.  I think we can close this once we make a new patch.
